app/views/OtherThingsToReportView.scala.html

```diff
@@ -0,0 +1,55 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.MovementReferenceNumber
+@import viewModels.components.InputCharacterCountViewModel._
+@import views.html.components.{Button, ErrorSummary, Heading, InputCharacterCount, InputYesNo}
+
+@this(
+    mainTemplate: MainTemplate,
+    formHelper: FormWithCSRF,
+    heading: Heading,
+    inputYesNo: InputYesNo,
+    errorSummary: ErrorSummary,
+    button: Button,
+    inputCharacterCount: InputCharacterCount
+)
+
+@(form: Form[_],mrn: MovementReferenceNumber, arrivalId: ArrivalId, maxLength: Int, mode: Mode)(implicit request: Request[_], messages: Messages)
+
+@mainTemplate(
+    title = messages("otherThingsToReport.title"),
+    showBackLink = true
+) {
+
+    @errorSummary(form.errors)
+
+    @formHelper(action = controllers.routes.OtherThingsToReportController.onSubmit(arrivalId, mode), Symbol("autoComplete") -> "off") {
+
+
+        @inputCharacterCount(
+          field = form("value"),
+          inputCharacterCountType = OrdinaryInputCharacterCount(
+              heading = messages("otherThingsToReport.title"),
```

Comment:
The messages are typically the same but I think this should be your `.heading` key